### ISSUE TITLE
Access to config/profiles.xml

### ISSUE THREAD

**Issue by cpy82** [jpspringall/serviio#2]
Hello jpspringall,

thank you for this project!

Could you please tell me how to get /serviio/config/profiles.xml or similar configuration files of serviio to edit them?

I would like to edit renderer profiles among other things: https://serviio.org/index.php?option=com_content&view=article&id=16

I had already tried to bind /opt/serviio/config, which did not work. But I also read that you removed the config volume.

Is there any way to edit the config files?

Thank you!

**Comment by jpspringall** [comment]
Hi,
Yes I removed the mount for the config as it is included in the downloaded zip, and is required to start the application.

If you are interested here is a link to the zip file that gets used during the docker image [Serviio Zip Install](http://download.serviio.org/releases/serviio-2.2.1-linux.tar.gz)

When you mount a volume to the path: /opt/serviio/config/
It basically means the config directory is empty as it is pointing at your host folder not the config folder from the downloaded zip folder.

However looking at the instructions in the link you supplied: [Editing renderer profiles](https://serviio.org/index.php?option=com_content&view=article&id=16)
I don't think the developer of serviio intendeds the profiles.xml or the applications-profile.xml to be edited directly.
I think they intended you to create a file called user-profiles.xml in the config directory.
It is possible to [mount a single file](https://stackoverflow.com/questions/42248198/how-to-mount-a-single-file-in-a-volume)

**You must ensure that the user-profiles.xml exists on your host system first**
Otherwise it will create a directory called user-profiles.xml
In my case it was just a case of adding the following to my docker compose under volumes:
\- /diskid/DLNA/Serviio/Config/user-profiles.xml:/opt/serviio/config/user-profiles.xml 

I also had to delete and recreate the docker container.

This is the [user-profiles.xml](https://github.com/jpspringall/serviio/files/7876456/user-profiles.zip) I used for testing purposes

Cheers

James

**Comment by cpy82** [comment]
Hi,
thank you for your quick reply! That helped me a lot!

I've been trying for months to get my older Philips Streamium audio players working again with internet radio stations via UPnP, since Philips has stopped managing it. Yesterday I discovered Serviio....

Unfortunately, the playback had not worked with this either. However, I came across the following post, hence my question about editing the profiles.xml: https://forum.serviio.org/viewtopic.php?f=5&t=29250

I have now solved it as follows:
mount a volume to the path: /opt/serviio/config/ and put the config files from the Serviio ZIP there on my host. Serviio installed and started without problems.

I was able to edit the profiles.xml for test purposes without any problems and remove the corresponding lines from the above mentioned article.

Playback via UPnP worked immediately afterwards!

I will now look at whether and how this could be implemented with a user-profiles.xml, then I do not have to change the profiles.xml. 

Thanks a lot!
